packages/mdx/src/mini-editor/editor-frame.tsx

```diff
@@ -167,7 +167,7 @@ function TabsContainer({
       )}
       {showFrameButtons ? <FrameButtons /> : <div />}
       {tabs.map(({ title, active, style }) => (
-        <div
+        <button
```

Comment:
Just a passing suggestion (maybe not something to implement here in this PR, but just a thought).

I would maybe suggest using an accessible UI foundation package for the editor (which is essentially a Tabs component, I think?). There are quite a few [keyboard interactions](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) (e.g. roving focus) that are a bit tricky to get right when rolling your own.

For comparison's sake, here are a few that I know of, off the top of my head:

- https://react-spectrum.adobe.com/react-aria/useTabList.html
- https://reach.tech/tabs
- https://reakit.io/docs/tab/
- https://www.radix-ui.com/docs/primitives/components/tabs

Reply:
valid aspect; I've opened a [new issue](https://github.com/code-hike/codehike/issues/344) for this, as I'm not necessarily a React expert and couldn't support on this end.